codespell_lib/data/dictionary.txt

```diff
@@ -29758,6 +29758,8 @@ scriopting->scripting
 scriopts->scripts
 scriopttype->scripttype
 scriping->scripting
+scripot->script
+scripots->scripts
```

Comment:
Maybe add `scripoting->scripting`?

Reply:
No hits in GitHub, only [109 Google hits](https://www.google.com/search?q=scripoting). But since `script` can be verb, we might add `scripoted`/`scripoting` as well.